src/lib/common/Geocoder.svelte

```diff
@@ -1,34 +1,30 @@
-<script lang="ts">
-    import GeocodingControl from "@maptiler/geocoding-control/svelte/GeocodingControl.svelte";
-    import type { MapController } from "@maptiler/geocoding-control/types";
-    import maplibregl from "maplibre-gl";
-    import { map } from "stores";
-    import { onMount } from "svelte";
-  
-    // TODO The position for the sketch page is very awkward, but there's
-    // seemingly no option to make the menu appear above the widget
-    export let position: "top-left" | "top-right" = "top-left";
-  
-    let mapController: MapController;
-  
-    // TODO HMR is broken
-    onMount(() => {
-      mapController = createMapLibreGlMapController($map, maplibregl);
+<script>
+  import { map } from "stores";
```

Comment:
Not saying this is the problem you hit, but this doesn't work here. `stores` is a common library in ATIP; you don't have it here. You have to plumb through the maplibre map object to this component differently. The top-level `MapLibre` component makes a `map` prop available to anything nested, so you could do `export let map;` here to receive it. Give https://learn.svelte.dev/tutorial/declaring-props a read

Reply:
Yes that makes sense. I wondered what `stores` was. Thanks Dustin!